subscription/src/test/java/org/killbill/billing/subscription/api/user/TestSubscriptionHelper.java

```diff
@@ -150,15 +150,20 @@ public DefaultSubscriptionBase createSubscription(final SubscriptionBaseBundle b
     }
 
     public DefaultSubscriptionBase createSubscription(final boolean noEvents, final SubscriptionBaseBundle bundle, final String productName, final BillingPeriod term, final String planSet) throws SubscriptionBaseApiException {
-        return createSubscription(noEvents, bundle, productName, term, planSet, null, null);
+        return createSubscription(noEvents, bundle, productName, term, planSet, null, null,null);
     }
 
     public DefaultSubscriptionBase createSubscription(final SubscriptionBaseBundle bundle, final String productName, final BillingPeriod term, final String planSet, final PhaseType phaseType, final LocalDate requestedDate)
             throws SubscriptionBaseApiException {
-        return createSubscription(false, bundle, productName, term, planSet, phaseType, requestedDate);
+        return createSubscription(false, bundle, productName, term, planSet, phaseType, requestedDate,null);
     }
+    
+    public DefaultSubscriptionBase createSubscription(final SubscriptionBaseBundle bundle, final String planName)
+            throws SubscriptionBaseApiException {
+        return createSubscription(false, bundle, null, null, null, null, null,planName);
+    }        
 
-    private DefaultSubscriptionBase createSubscription(final boolean noEvents, @Nullable final SubscriptionBaseBundle bundle, final String productName, final BillingPeriod term, final String planSet, final PhaseType phaseType, final LocalDate requestedDate)
+    private DefaultSubscriptionBase createSubscription(final boolean noEvents, @Nullable final SubscriptionBaseBundle bundle, final String productName, final BillingPeriod term, final String planSet, final PhaseType phaseType, final LocalDate requestedDate, final String planName)
```

Comment:
Could we group all the catalog info in the signature, e.g `productName`, `term`, `planSet`, `planName` and then the `requestedDate` ?

Reply:
Fixed